ext/native-decls/SetAudioSubmixOutputVolumes.md

```diff
@@ -0,0 +1,24 @@
+---
+ns: CFX
+apiset: client
+game: gta5
+---
+## SET_AUDIO_SUBMIX_OUTPUT_VOLUMES
+
+```c
+void SET_AUDIO_SUBMIX_OUTPUT_VOLUMES(int submixId, int outputSubmixId, float frontLeftVolume, float frontRightVolume, float rearLeftVolume, float rearRightVolume, float channel5Volume, float channel6Volume);
+```
+
+Sets the volumes for the sound channels in a submix effect.
+Values can be between 0.0 and 1.0.
+Channel 5 and channel 6 are not used in voice chat but are believed to be center and LFE channels.
+
+## Parameters
+* **submixId**: The submix.
+* **outputSubmixId**: The output submix. Use 0 for the master game submix.
```

Comment:
Is this actually correct, and is it not an output slot as I suspected in initial documentation? (i.e. 0 will be the first AddOutput call, 1 the second, etc.)

Reply:
Yes I misunderstood again, this is exactly as you are saying

Full repro code I used to validate this:

local radioSubmix = CreateAudioSubmix('Radio')
SetAudioSubmixEffectRadioFx(radioSubmix, 0)
SetAudioSubmixEffectParamInt(radioSubmix,   0, `enabled`,     1)
SetAudioSubmixEffectParamFloat(radioSubmix, 0, `freq_low`,    202.0)
SetAudioSubmixEffectParamFloat(radioSubmix, 0, `freq_hi`,     6000.0)
SetAudioSubmixEffectParamFloat(radioSubmix, 0, `rm_mod_freq`, 0.0)
SetAudioSubmixEffectParamFloat(radioSubmix, 0, `rm_mix`,      0.0)
SetAudioSubmixEffectParamFloat(radioSubmix, 0, `fudge`,       5.0)
SetAudioSubmixEffectParamFloat(radioSubmix, 0, `o_freq_lo`,   141.0)
SetAudioSubmixEffectParamFloat(radioSubmix, 0, `o_freq_hi`,   6000.0)
AddAudioSubmixOutput(radioSubmix, 0)

local dummySubmix = CreateAudioSubmix('Dummy')
SetAudioSubmixEffectRadioFx(dummySubmix, 0)
SetAudioSubmixEffectParamInt(dummySubmix,   0, `enabled`,     1)
SetAudioSubmixEffectParamFloat(dummySubmix, 0, `freq_low`,    202.0)
SetAudioSubmixEffectParamFloat(dummySubmix, 0, `freq_hi`,     6000.0)
SetAudioSubmixEffectParamFloat(dummySubmix, 0, `rm_mod_freq`, 0.0)
SetAudioSubmixEffectParamFloat(dummySubmix, 0, `rm_mix`,      0.0)
SetAudioSubmixEffectParamFloat(dummySubmix, 0, `fudge`,       50.0)
SetAudioSubmixEffectParamFloat(dummySubmix, 0, `o_freq_lo`,   141.0)
SetAudioSubmixEffectParamFloat(dummySubmix, 0, `o_freq_hi`,   6000.0)

local radioSubmix2 = CreateAudioSubmix('Radio2')
SetAudioSubmixEffectRadioFx(radioSubmix2, 0)
SetAudioSubmixEffectParamInt(radioSubmix2,   0, `enabled`,     1)
SetAudioSubmixEffectParamFloat(radioSubmix2, 0, `freq_low`,    402.0)
SetAudioSubmixEffectParamFloat(radioSubmix2, 0, `freq_hi`,     6000.0)
SetAudioSubmixEffectParamFloat(radioSubmix2, 0, `rm_mod_freq`, 0.0)
SetAudioSubmixEffectParamFloat(radioSubmix2, 0, `rm_mix`,      0.0)
SetAudioSubmixEffectParamFloat(radioSubmix2, 0, `fudge`,       00.0)
SetAudioSubmixEffectParamFloat(radioSubmix2, 0, `o_freq_lo`,   141.0)
SetAudioSubmixEffectParamFloat(radioSubmix2, 0, `o_freq_hi`,   6000.0)
AddAudioSubmixOutput(radioSubmix2, dummySubmix)
AddAudioSubmixOutput(radioSubmix2, radioSubmix)

MumbleSetSubmixForServerId(1, radioSubmix2)

--SetAudioSubmixOutputVolumes(radioSubmix2, 0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0) -- will not mute
--SetAudioSubmixOutputVolumes(radioSubmix2, 1, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0) -- will mute

--SetAudioSubmixOutputVolumes(radioSubmix, 0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0) -- will mute
--SetAudioSubmixOutputVolumes(radioSubmix, 1, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0) -- will not mute